reana_client/utils.py

```diff
@@ -206,7 +204,9 @@ def _validate_reana_yaml(reana_yaml):
         raise e
     except ValidationError as e:
         logging.info("Invalid REANA specification: {error}".format(error=e.message))
-        raise e
+        raise REANAValidationError(
+            '==> ERROR:  The `reana.yaml` is not valid: "{0}".'.format(e.message)
+        )
```

Comment:
Couple of things:
- It'd be nice to keep the same colors for the error messages and rely on `display_message()` instead of appending ourselves the `"==> ERROR:"` prefix
- Error code is 0 when failing

![image](https://user-images.githubusercontent.com/8863238/135454097-9d2b6a46-05b5-4f6d-aa8f-35047d74c684.png)